<?xml version="1.0"?>
<launch>

	<node pkg="bluerov2_dobmpc" type="bluerov2_dob_node" name="bluerov2_dob_node" output="screen">
        <rosparam command="load" file="$(find bluerov2_dobmpc)/config/gazebo_param.yaml" />
		<!--<param name="ref_traj" type = "string" value="$(find bluerov2_mpc)/traj/circle.txt" />-->
	</node>

</launch>